src/api/users/helpers/aggregations/user.js

```diff
@@ -1,61 +1,273 @@
-export const userAggregation = [
+export const userAggregation = (currentDateTime = new Date()) => [
```

Comment:
I guess if we can test it, its probably fine but the aggregation (like most mongo aggergations) is pretty hard to follow.
Wondering if it would be clearer/easier to test etc if it were done in code?